__tests__/routes/applications.test.js

```diff
@@ -219,4 +222,123 @@ describe('Application Router', () => {
       });
     });
   });
+
+  describe('[POST] /application/new/:role', () => {
+    const validMentee = {
+      city: 'Portland',
+      convictions: 'String Data',
+      country: 'United States',
+      email: 'joe2@mageddon.com',
+      first_name: 'Joe',
+      formerly_incarcerated: true,
+      job_help: true,
+      last_name: 'Mageddon',
+      low_income: true,
+      other_info: 'No',
+      pair_programming: false,
+      referred_by: 'coworker',
+      state: 'Oregon',
+      tech_stack: 'Design UI/UX',
+      underrepresented_group: false,
+      validate_status: 'pending',
+      is_active: false,
+      in_project_underdog: false,
+      profile_id: '4e157cae-af53-4645-a865-53a62bcfc08e',
+    };
+
+    const invalidMentee = {
+      email: "joe2@mageddon.com",
+      first_name: "Joe",
+      last_name: "Mageddon",
+      validate_status: 'pending',
+    }
+
+    const validMentor = {
+      city: 'Hartford',
+      commitment: false,
+      country: 'United States',
+      current_company: 'AWS',
+      current_position: 'Engineering Manager',
+      email: 'gilly@woisley.com',
+      first_name: 'Gilly',
+      job_help: true,
+      last_name: 'Woisley',
+      other_info: 'Kindness',
+      pair_programming: true,
+      referred_by: 'facebook',
+      state: 'Connecticut',
+      tech_stack: [
+        'Backend',
+        'Frontend',
```

Comment:
Endpoint tests for failure scenarios are able to correctly reject imporperly formatted payloads, however they inaccurately return error code 500 despite an error message that referebces an error code 422.

This is likely due to the configuration of the error handler, which defaults to 500 when an error occurs with the err.status being defined - see `api/middleware/handleError.js`